README.rst

```diff
@@ -1,13 +1,17 @@
 User documentation
 ==================
 
+.. image:: https://codecov.io/gh/spacetelescope/spherical_geometry/branch/master/master/graphs/badge.svg
```

Comment:
Why not add the CI badge too, while you are at it?

Reply:
I added a CI badge.